language/diem-vm/src/diem_transaction_executor.rs

```diff
@@ -268,13 +265,10 @@ impl DiemVM {
         gas_status
             .charge_intrinsic_gas(txn_data.transaction_size())
             .map_err(|e| e.into_vm_status())?;
+
+        // TODO: tracing to diem logger
```

Comment:
What's this TODO about? Is it something that needs to be addressed before this lands?

Reply:
Oh, it was a legacy comment to remind myself to set the global subscriber for tracing. Given now that we're doing it properly in the diem-logger, this comment can be deleted safely.